kubetest/main.go

```diff
@@ -742,7 +742,7 @@ func prepare(o *options) error {
 		if err := prepareGcp(o); err != nil {
 			return err
 		}
-	case "aws":
+	case "aws", "kube-aws":
```

Comment:
Mentioned offline, but I think we should just switch the jobs to `--provider=aws` and remove the .env setting. `kubetest` already sets `KUBERNETES_CONFORMANCE_PROVIDER` to aws.